roles/obol_charon/templates/zypher/mainnet/systemd.service.j2

```diff
@@ -21,8 +20,6 @@ ExecStart={{ obol_charon_bin_dir }}/{{ service_identifier }} run \
   --monitoring-address 0.0.0.0:{{ obol.charon.ports.metrics }} \
   --builder-api true \
   --feature-set-enable eager_double_linear,consensus_participate \
-  # --loki-addresses http://loki:3100/loki/api/v1/push \
-  # --loki-service charon \
```

Comment:
why remove it ? we will need to push logs and metrics to their grafana, just like we do in supra